### PR TITLE
bug(issue): Support explicit transitions for issues

### LINKED ISSUE #38: Transition name is ignored when passed to `jira i <name> --transition`
I would expect the following command to transition my issue to "In Progress"

```
jira i OHANA-1105 --transition "In Progress"
```

However, it does not... I am just presented with a list of transitions and can select one.

### PR DESCRIPTION
**Pull request by dancrumb**
The CLI was ignoring transition names when it came to
`jira i XXX --transition YYY`. This fixes that.

Fixes #38

### CODE REVIEW

File: src/issues.js [hunk 1]
```diff
@@ -435,25 +435,40 @@ export default class JiraIssues {
...
-  async makeTransition( issueId ){
+  async makeTransition( issueId, transitionName ){
 
     const transitions = await this.getIssueTransitions(issueId);
-    const _this = this;
 
-    if ( transitions.length == 1 ) {
+    let transitionObj;
+
+    if (transitionName) {
```
Comment (miguelmich) [inline src/issues.js]: @dancrumb excellent! just one little detail, can you please add an additional check here, the transitionName is returning `true` if there is no parameter, so we need to introduce something like `if (transitionName && transitionName !== true) {` what do you think?

**Review by miguelmich — CHANGES_REQUESTED** [review]

File: src/issues.js (hunk 1 above)
Comment (dancrumb) [reply src/issues.js]: I think this will do it

```suggestion
    if (typeof transitionName === 'string' && transitionName.length > 0) {
```

**Comment by rdourado** [comment]
Why wasn't this released?

**Comment by dancrumb** [comment]
Looks like the last release was 6 months prior to this merge :/

**Pull request merged**